introduction-to-programming.md

```diff
@@ -106,37 +124,39 @@ From now on work in separate HTML files per exercise.
   ```
 
 ### Exercise 04
-- Write the numbers from 1 to 50 to the console
+- Create an array of the names of your neighbours at the table (use at least 4)
+- Using `forEach` write all the names to the console
+- Using `forEach` greet each of them individually like: `Ahoj, Aze`
+- Add yourself to the array and check greeting yourself
```

Comment:
it is not clear to me whether should we use `push` or just modify the array

Reply:
I guess it doesn't matter, though my idea was just to modify the array at this point